spec/lib/jobs_spec.rb

```diff
@@ -34,6 +34,19 @@
         expect(search).to be_nil
       end
     end
+
+    context 'when the search is within the default radius, but in a different city' do
```

Comment:
@peggles2 , this doesn't necessarily test the radius, because this spec would pass if there were any jobs available in Baltimore. I would expect to see the `it 'searches USAJOBS with the correct params` spec updated with the expected radius.